ProjectLighthouse.Localization/Moderation.resx

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<root>
+    <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+        <xsd:element name="root" msdata:IsDataSet="true">
+            
+        </xsd:element>
+    </xsd:schema>
+    <resheader name="resmimetype">
+        <value>text/microsoft-resx</value>
+    </resheader>
+    <resheader name="version">
+        <value>1.3</value>
+    </resheader>
+    <resheader name="reader">
+        <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <resheader name="writer">
+        <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    </resheader>
+    <data name="banned_heading" xml:space="preserve">
+        <value>Account Banned</value>
+    </data>
+    <data name="banned_explain" xml:space="preserve">
+        <value>Your account has been banned due to inappropriate, offensive, or violent behavior.</value>
+    </data>
+    <data name="case_created" xml:space="preserve">
```

Comment:
I'm just going off how other services usually handle bans, but you usually aren't told when you were banned. Generally, they just have the reason and when it expires.

Reply:
I'd have to check the current ban string but I believe it mentions something of the same aspect.

Reply:
I think it'd allow the user to have a general scope of their punishment length, opting to keep it.